app/views/users/index.html.erb

```diff
@@ -7,7 +7,7 @@
     </div>
     <div class="lg-12-3">
       <% unless current_user %>
-        <%= link_to user_omniauth_authorize_path(:github), class: "btn btn-github" do %>
```

Comment:
@enderahmetyurt valla denemek için üşenmedim yeni github hesabı bile açıp rubyturkiye.org'a kaydoldum. şu testaccountfoo ismiyle kayıt olan benim ve gördüğün gibi Github URL'i görünmüyor.